store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -192,4 +192,35 @@ public void modifyServerProperties(VolumeInfo volInfo) throws ServiceException,
             throw e;
         }
     }
+
+    /**
+     * Validate global bucket ID by reading LDAP atrribute
+     * @param globalS3BucketId
+     * @returns true if "global bucket ID" is valid
+     * @throws ServiceException, JSONException
+     */
+    public Boolean validateGlbBucketID(String globalS3BucketId) throws ServiceException, JSONException {
+        Boolean isValid = false;
+        try {
+            // step 1: Fetch current JSON state object and current JSON state array
+            String globalExternalStoreConfigJson = provisioning.getConfig().getGlobalExternalStoreConfig();
+            JSONObject currentJsonObject = new JSONObject(globalExternalStoreConfigJson);
```

Comment:
`currentJsonObject` & `currentJsonArray` needs having mode meaning full name in regards to feature or the object its holding

Reply:
Done